apps/kampus/app/pano/features/post-list/MoreOptions.tsx

```diff
@@ -54,7 +56,15 @@ export const MoreOptionsDropdown = ({ post, shareUrl }: Props) => {
       </DropdownMenuTrigger>
       <DropdownMenuContent>
         {ownerItems}
-        <DropdownMenuItem>Adresi kopyala</DropdownMenuItem>
+        <DropdownMenuItem
+          onSelect={() => {
+            toast({
+              description: "link kopyalandı",
+            });
+          }}
+        >
+          Adresi kopyala
```

Comment:
```suggestion
          Linki kopyala
```

Hangi kelimeyi kullanmamiz gerektigine tam karar veremiyorum, var mi oneriniz? @usirin @Ketcap

Reply:
Linki kopyala daha mantikli, let's go